SU-SWS/stanford_profile
# D8CORE-3715 Updated field encrypt to 3.0.0

# DO NOT MERGE

# Summary
- Updated field encrypt module to 3.0.0

# Need Review By (Date)
- :shrug: 

# Urgency
- low

# Steps to Test
1. using the 8.x or 9.x branches, add some values to the person importer credentials field.
2. checkout just the update file from this PR: `git checkout D8CORE-3715 -- stanford_profile.post_update.php`
   - What we will probably do is run the functions via `drush eval` just before doing a code deployment because we have to decrypt and uninstall the module before we update it.
4. `drush updb`
5. require this branch `composer require su-sws/stanford_profile:dev-D8CORE-3715`
6. `blt drupal:update` or `drush deploy`
7. view the person importer config page
8. verify your values are still as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
